Gemfile

```diff
@@ -23,8 +23,9 @@ gem 'rails', '5.2.1'
 gem 'responders'
 gem 'sidekiq', '= 5.2.3'
 gem 'sinatra', :require => nil
-gem 'supplejack_common', git: 'https://github.com/DigitalNZ/supplejack_common.git', tag: 'v2.5'
+gem 'supplejack_common', git: 'https://github.com/DigitalNZ/supplejack_common.git', branch: 'tw/s3-bucket-emu-harvest'
```

Comment:
You'll need to change this to the new tag of SJ common before you merge the worker.